2020-02-Spring/3_trees/README.md

```diff
@@ -70,21 +70,91 @@ Go to [Solution](#s2)   [Top](#top)
 <!-- Don't remove -->
 <a name="p3"/>
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Recover Binary Search Tree
 
-Source: TODO :bug:
+[LeetCode](https://leetcode.com/problems/recover-binary-search-tree/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Two elements of a binary search tree (BST) are swapped by mistake.
+
+Recover the tree without changing its structure.
+
+**Follow-up**: Can you devise a constant space solution?
```

Comment:
Some folks might not know what constant space is, so it would be a good idea to either explain it in the CTCI session or explaining it right here.

Reply:
👍